app/operations/workflows/retire_subjects.rb

```diff
@@ -6,11 +6,12 @@ class RetireSubjects < Operation
     array :subject_ids, default: [] do
       integer
     end
+    string :retirement_reason, default: nil
```

Comment:
Are we controlling the values that can be set here, what if i pass in `retirement_reason: "not_a_reason"`? we haven't spec'd this scenario and i assume it'll be set to nil by default. I think this is fine but we should spec this expectation to make sure that behaviour persists OR control the input values for this attribute.